### PR TITLE
Add a area level for regions

### PR DESCRIPTION
**Pull request by WangLiNaruto**
[#181 ](url)

### CODE REVIEW

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/resource/Area.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import lombok.Data;
+
+/**
+ * Define the area of the region.
+ */
+@Data
+public class Area {
+
+    @NotNull
+    @Schema(description = "The name of the area")
```
Comment (iskey) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/resource/Area.java:21]: Also, adding @NotBlank is better?

**Comment by iskey** [comment]
Please fix all the OCL examples with the same.

**Comment by davidinux** [comment]
@WangLiNaruto it appears you haven't either completed ECA process or using an email that is not linked to your Eclipse Foundation contributor profile, hence the merge is flagged as blocked.

**Review by davidinux — APPROVED** [review]
LGTM

**Comment by niuzhenguo** [comment]
All contributors must sign the ECA, pls follow this https://www.eclipse.org/projects/handbook/#contributing-eca

**Review by iskey — CHANGES_REQUESTED** [review]
Please fix all the examples.

**Comment by iskey** [comment]
As we decide to make the `area` a field of the `Region`, as #210 does, I am going to close this PR now.

**Pull request closed without merging**